config/katello.migrations/200123161606-enable-pulpcore.rb

```diff
@@ -0,0 +1,5 @@
+cls = 'foreman_proxy::plugin::pulp'
```

Comment:
Where possible, I prefer, for readability, not to use shortened names for things. So something like `puppet_class` as the variable name is more expressive.

Reply:
Probably right. On a mostly unrelated note, I've been thinking about a helper in kafo to simplify this pattern but haven't been able to come up with one that actually makes it clear.